login-workflow/src/new-architecture/contexts/AuthContext/provider.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * @packageDocumentation
+ * @module RegistrationWorkflowContextProvider
+ */
+
+import React, { useEffect } from 'react';
+import { AuthContextProviderProps } from './types';
+import { AuthContext } from './context';
+import { I18nextProvider } from 'react-i18next';
+import { i18nAuthInstance } from './i18nAuthInstance';
+
+export const AuthContextProvider: React.FC<React.PropsWithChildren<AuthContextProviderProps>> = (props) => {
+    const { language, i18n = i18nAuthInstance } = props;
+
+    useEffect(() => {
+        void i18n.changeLanguage(language);
+    }, [i18n, language]);
+
+    return (
+        <I18nextProvider i18n={i18n}>
+            <AuthContext.Provider value={props}>{props.children}</AuthContext.Provider>
```

Comment:
setting `value` to props is inadvertently including the children in the context (which we do not want to do). You'll need to destructure children out of the variable, or manually set up the 4 variables we want to include instead of spreading all of props.

e.g.,
```
const { children, ...authContextProps } = props;
const { language, i18n = i18nAuthInstance } = authContextProps;
...
<AuthContext.Provider value={authContextProps}>{children}</AuthContextProvider>
```

Reply:
Check this in the registration context provider too...I'm not sure if I caught this there.